torch/fx/experimental/sym_node.py

```diff
@@ -83,7 +83,7 @@ def __init__(
         expr,
         shape_env,
         pytype,
-        hint: Optional[Union[int, float, bool]],
+        hint: Optional[Union[int, float, bool, torch.SymInt]],
```

Comment:
i might have already asked this and can't find the comment: it sort of makes sense to me that the hint can be a `SymInt` because nested_ints are actual `SymInts` at runtime (not just a compile time constant). But why do we need this now vs. we didn't need this change before this PR? Or is this change independent

Reply:
Ah hint has already been able to be nested int prior to this PR, this just updates the annotation. Sorry for the noise!